### DISCUSSION TITLE
Рефакторинг кода сигналки

### DISCUSSION THREAD

**Discussion by glebqip** [metrostroi-repo/MetrostroiAddon#207] [General]
#106 (кастомные модели светофоров)
#18 (оптимизация)
Код был написан очень давно

**Comment by CralixRaev** [comment]
Готов предоставить модели сигнализации от команды маяковского, что бы сигналку переписывать сразу с нормальными моделями
![image](https://user-images.githubusercontent.com/32234868/104164188-9f0f8c80-5408-11eb-87b3-77a560af1db0.png)
![image](https://user-images.githubusercontent.com/32234868/104164240-b0589900-5408-11eb-9576-1cf793b8ae06.png)

**Comment by kosmik641** [comment]
> Готов предоставить модели сигнализации от команды маяковского, что бы сигналку переписывать сразу с нормальными моделями

Причем тут модели, если в первую очередь проблема в скриптах, а не в моделях? Крикет засовывал модели без полигонов, и существенной прибавки FPS не произошло.

**Comment by glebqip** [comment]
Новые модели это хорошо, но сначала надо разобраться с скриптами. Опять же, ещё надо нормально разобраться с #106

**Comment by CralixRaev** [comment]
> > Готов предоставить модели сигнализации от команды маяковского, что бы сигналку переписывать сразу с нормальными моделями
> 
> Причем тут модели, если в первую очередь проблема в скриптах, а не в моделях? Крикет засовывал модели без полигонов, и существенной прибавки FPS не произошло.

если мы собираемся делать сигналку с новыми модельками, стоить сразу писать её с новыми модельками, т.к. многое может отличаться, как допустим работа и расположение маршрутных указателей

**Comment by kosmik641** [comment]
> > > Готов предоставить модели сигнализации от команды маяковского, что бы сигналку переписывать сразу с нормальными моделями
> > 
> > Причем тут модели, если в первую очередь проблема в скриптах, а не в моделях? Крикет засовывал модели без полигонов, и существенной прибавки FPS не произошло.
> 
> если мы собираемся делать сигналку с новыми модельками, стоить сразу писать её с новыми модельками, т.к. многое может отличаться, как допустим работа и расположение маршрутных указателей

1. Кто мы, если список скриптеров и моделлеров для фиксов утвержден?
2. Issue по поводу кастомных светофоров уже создан, а значит будет решен. Зачем так упорно продвигать свои модели? В первую очередь репозиторий создан не для нового контента, а для фикса багов.

**Comment by CralixRaev** [comment]
> > > > Готов предоставить модели сигнализации от команды маяковского, что бы сигналку переписывать сразу с нормальными моделями
> > > 
> > > 
> > > Причем тут модели, если в первую очередь проблема в скриптах, а не в моделях? Крикет засовывал модели без полигонов, и существенной прибавки FPS не произошло.
> > 
> > 
> > если мы собираемся делать сигналку с новыми модельками, стоить сразу писать её с новыми модельками, т.к. многое может отличаться, как допустим работа и расположение маршрутных указателей
> 
> 1. Кто мы, если список скриптеров и моделлеров для фиксов утвержден?
> 2. Issue по поводу кастомных светофоров уже создан, а значит будет решен. Зачем так упорно продвигать свои модели? В первую очередь репозиторий создан не для нового контента, а для фикса багов.

1. как говорится, кто мы то б**ть, я тут один
2. хотелось бы не только пофиксить баги, а также внести новый контент и улучшить то, что есть сейчас. я не продвигаю свои модели, а предлагаю то, что намного лучше чем существующие светофоры. пока что светофоры мусьена находятся на уровне 2017 года, хотя на дворе 2021. а с кастомными светофорами у меня очень много вопросов, т.к. для разных целей понадобится менять не только модели и координаты, а так же и логику многих вещей, но это уже ваша проблема)
